config/app_config.yml.sample

```diff
@@ -109,6 +109,12 @@ defaults: &defaults
     # base_url: 'https://common-data.cartodb.com'
     format: 'shp'
     generate_every: 86400
+  fdw:
```

Comment:
It seemed more sensible to provide this configuration here, rather than pass it with every request via the Data Import API. This doesn't seem all that scalable though, have you guys experimented with other ways to provide this config?